### PR TITLE
Docs and templates

### PR DESCRIPTION
**Pull request by silverdaz**
Github templates.... for Jocke

.... and the docs I had forgotten to push..... for Oscar.

I know, I'm doing 2 things are once. 
But they are small things.

### CODE REVIEW

**Comment by silverdaz** [comment]
Connected to [card](https://github.com/orgs/EGA-archive/projects/3#card-19472111)

File: src/mq-config.c [hunk 1]
```diff
@@ -99,7 +98,7 @@ readconfig(FILE* fp, const char* cfgfile, char* buffer, size_t buflen)
   /* Default config values */
   mq_options->heartbeat = MQ_HEARTBEAT;
   mq_options->connection_opened = 0; /* not opened yet */
-  mq_options->ssl = MQ_ENABLE_SSL;
```
Comment (jbygdell) [inline src/mq-config.c:102]: Doesn't this effectively disable SSL connection for the local MQ instance?

Comment (silverdaz) [reply src/mq-config.c:102]: No, it's the default value of the compilation flag.
When you compile, I gave the option to define what the default value was, via a preprocessor definition.
In the end, I didn't use the compilation flag.
The ssl option is still there, and configurable.

**Review by omllobet — APPROVED** [review]
The ssl enable option is obtained from the connection url

File: src/mq-config.c (hunk 1 above)
Comment (jbygdell) [reply src/mq-config.c:102]: OK.
This specific commit does not really belong in this PR, but hey it's Friday.

**Review by jbygdell — APPROVED** [review]

**Pull request merged**